### PR TITLE
Stronger prompt for recording identity anchor

### PR DESCRIPTION
**Pull request by frederikrothenberger**
Added stronger wording on the prompt to record the Identity Anchor.

Without the Identity Anchor identities can only be recovered with a recovery seed phrase. If the Identity Anchor is lost (e.g. by deleting the browser data) before creating a seed phrase, the identity itself is lost.

### CODE REVIEW

File: src/frontend/src/flows/displayUserNumber.ts [hunk 1]
```diff
@@ -1,12 +1,24 @@
...
+     Internet Identity or to add additional devices.
     </p>
+    <div class="nagBox">
+      <div class="nagIcon">${warningIcon}</div>
+      <div class="nagContent">
+        <div class="nagTitle">Record Your Identity Anchor</div>
+        <div class="nagMessage">
+          Do <b>NOT</b> forget to save this Identity Anchor. Save a backup on a
+          storage medium and write it down.</br>
+          Note: It is advised to keep Identity Anchors secret in order to stay anonymous.
```
Comment (nmattia) [inline src/frontend/src/flows/displayUserNumber.ts:18]: What do you mean by "stay anonymous"? Anchors are never shared when authenticating

**Review by nmattia — COMMENTED** [review]
One comment; can you also share some screenshots?

File: src/frontend/src/flows/displayUserNumber.ts (hunk 1 above)
Comment (frederikrothenberger) [reply src/frontend/src/flows/displayUserNumber.ts]: Ah, right.

**Comment by frederikrothenberger** [comment]
> One comment; can you also share some screenshots?

Sure
Screenshot 2021-11-30 at 17 09 51

**Review by nmattia — APPROVED** [review]
Looks great! What do you think about moving the "You will need it later..." bit down with the warning? If we're gonna warn, better give a reason!

File: src/frontend/src/flows/displayUserNumber.ts [hunk 2]
```diff
@@ -1,12 +1,23 @@
...
-      Please record your new Identity Anchor. You will need it later to use
+      Your new Identity Anchor has been created. You will need it later to use
       Internet Identity or to add additional devices.
     </p>
+    <div class="nagBox">
+      <div class="nagIcon">${warningIcon}</div>
+      <div class="nagContent">
+        <div class="nagTitle">Record Your Identity Anchor</div>
+        <div class="nagMessage">
+          Do <b>NOT</b> forget to save this Identity Anchor. Save a backup on a
```
Comment (nmattia) [inline src/frontend/src/flows/displayUserNumber.ts:16]: I think the "Do **NOT** forget" bit is a bit strong, but that's just my 2cts

**Comment by frederikrothenberger** [comment]
@nmattia: Thanks for the feedback. This is what it looks now:
Screenshot 2021-12-01 at 08 40 54

File: src/frontend/src/flows/displayUserNumber.ts [hunk 3]
```diff
@@ -1,12 +1,21 @@
...
-    </p>
+    <p>Your new Identity Anchor has been created.</p>
+    <div class="nagBox">
+      <div class="nagIcon">${warningIcon}</div>
+      <div class="nagContent">
+        <div class="nagTitle">Record Your Identity Anchor</div>
+        <div class="nagMessage">
+          Do not forget to save this Identity Anchor. Save a backup on a storage
+          medium and write it down. You will need it later to use Internet
+          Identity or to add additional devices.
```
Comment (robin-kunzler) [inline src/frontend/src/flows/displayUserNumber.ts:15]: Could we even strengthen the text a bit more and say e.g. the following? 
```suggestion
          Please record your new Identity Anchor. Keep a backup on a storage
          medium and write it down. You will need it later to use Internet
          Identity or to add additional devices. If you lose your Identity Anchor, 
          you will no longer be able to use it to authenticate to dApps and you 
          will lose access to them. 
```

Comment (frederikrothenberger) [reply src/frontend/src/flows/displayUserNumber.ts:15]: Hm, I'm unsure about the "...you will lose access to them.". You can still access dApps, just not with that identity.

Maybe like this?

> Please record your new Identity Anchor. Keep a backup on a storage medium and write it down. You will need it later to use Internet Identity or to add additional devices. If you lose your Identity Anchor, you will no longer be able to use this identity to authenticate to dApps and you will lose access to the associated profiles and data.

Comment (robin-kunzler) [reply src/frontend/src/flows/displayUserNumber.ts:15]: Sounds great, thanks!

File: src/frontend/src/flows/displayUserNumber.ts [hunk 4]
```diff
@@ -1,12 +1,24 @@
...
+      <div class="nagIcon">${warningIcon}</div>
+      <div class="nagContent">
+        <div class="nagTitle">Record Your Identity Anchor</div>
+        <div class="nagMessage">
+          Please record your new Identity Anchor. Keep a backup on a storage
+          medium and write it down. You will need it later to use Internet
+          Identity or to add additional devices. If you lose your Identity
+          Anchor, you will no longer be able to use this identity to
+          authenticate to dApps and you will lose access to the associated
+          profiles and data.
```
Comment (nmattia) [inline src/frontend/src/flows/displayUserNumber.ts:18]: Nitpick: this could suggest that II has profiles and data about the user

Comment (frederikrothenberger) [reply src/frontend/src/flows/displayUserNumber.ts]: Done

**Pull request merged**